core/templates/domain/collection/collection-rights-backend-api.service.ts

```diff
@@ -111,7 +111,8 @@ export class CollectionRightsBackendApiService {
   /**
    * Gets a collection's rights, given its ID.
    */
-  fetchCollectionRights(collectionId: string): Promise<CollectionRights> {
+  async fetchCollectionRightsAsync(collectionId: string):
```

Comment:
This function is used in [collection-editor-state.service.ts](https://github.com/oppia/oppia/blob/develop/core/templates/pages/collection-editor-page/services/collection-editor-state.service.ts#L85) but the name has not been changed there. Can you check the usages for other functions as well?
The way I do it is I use the global search in my text editor (VSCode) and that usually lists out all the usages. I copy the name of the function (in this case "fetchCollectionRights") and add a dot before the name (so ".fetchCollectionRights") and search for it.

Reply:
![image](https://user-images.githubusercontent.com/68547101/97782980-c3eeb480-1bba-11eb-8718-e39a7c56a069.png)
 After I do this change- i reccieve the following error-

Reply:
After I change fetchCollectionRights to fetchcollectionRightAsync i recieve the following error-

Reply:
![image](https://user-images.githubusercontent.com/68547101/97783080-4ecfaf00-1bbb-11eb-9eed-cd168ee73d67.png)

Reply:
I have done the following changes-
In collection-editor-state.serice.ts
![image](https://user-images.githubusercontent.com/68547101/97783116-8f2f2d00-1bbb-11eb-884f-433113bc544a.png)
In  collection-rights-backend api service-
![image](https://user-images.githubusercontent.com/68547101/97783162-f220c400-1bbb-11eb-8e95-5c86e1486ca5.png)